src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4821onevariableperline/InputFormattedOneVariablePerDeclaration.java

```diff
@@ -0,0 +1,95 @@
+package com.google.checkstyle.test.chapter4formatting.rule4821onevariableperline;
+
+/** Some javadoc. */
+public class InputFormattedOneVariablePerDeclaration {
+  int mnp, efg; // violation 'Each variable declaration must be in its own statement.'
```

Comment:
looks like google formatter does not target splitting of variables to multiple lines
https://github.com/google/google-java-format/issues?q=is%3Aissue+variable+is%3Aopen

@Zopsss , please create issue on them to clarify is it a gap or it is our misinterpretation of style guide.

Reply:
it cannot. The formatter works only on formatting things like fixing indentation, whitespaces, etc it cannot create new variables just like it cannot fix invalid javadoc position. Just asking your confirmation again, should we create an issue for this?

Reply:
Please create issue on them, to let us have a feedback of this is point that they can not do and people should rely on checkstyle to highlight it.

Reply:
they already have issue on this: https://github.com/google/google-java-format/issues/1009 the issue is quite old but it seems inactive

Reply:
Cushon's reply on issue: https://github.com/google/google-java-format/issues/104#issuecomment-272338280

his replies looks similar to https://github.com/checkstyle/checkstyle/pull/15587#discussion_r1739103569, it is difficult ( or might be impossible ) for them to implement this feature

Reply:
this is exact match to problem that we see https://github.com/google/google-java-format/issues/104

ok, lets move forward. One day it will be fixed in formatter until that time users can rely on checkstyle to highlight a problems and user should fix it manually.